docs/model_developers/modules/handler.rst

```diff
@@ -0,0 +1,174 @@
+South Bound Integration
+***********************
+
+The inmanta orchestrator comes with a set of integrations with different platforms (see: :ref:`ref-modules`).
+But it is also possible to develop your own south bound integrations. 
```

Comment:
```suggestion
But it is also possible to develop your own south bound integrations.
```
There are a lot of these in this file. Perhaps run `sed -i 's/\s\+$//' docs/model_developers/modules/handler.rst`?

Reply:
thx, done